daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -1592,6 +1592,7 @@ message DefTemplate {
   message Implements {
     TypeConName interface = 1;
     InterfaceInstanceBody body = 2;
+    Location location = 3;
```

Comment:
Could you add a comment explaining they are ignored by Scala (I was not aware of that and quit annoyed by that) and they have been introduced latter than `Co`/`Implements` ? 

This would be probably very useful if we decide to handle this field on the scala side..

Reply:
Sure! Though I have sadly already merged, whats the expected set of actions here? A new PR? Reopen this one with the changes?

Reply:
I'm working on another PR rn anyway, so I could add them in that one and link back here?